lib/daru/io/importers/linkages/plaintext.rb

```diff
@@ -0,0 +1,23 @@
+require 'daru'
+
+module Daru
+  class DataFrame
+    class << self
+      # Read the database from a plaintext file. For this method to work,
+      # the data should be present in a plain text file in columns. See
+      # spec/fixtures/bank2.dat for an example.
+      #
+      # == Arguments
+      #
```

Comment:
Great work done so far! It would be good if you follow the more popular yard documentation style ([see here](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md)) because it is easy to read and write and would be pain to change if we decide to do so in the future. Also I think you forgot to add "yard" as dependency in your gemspec file.